x-pack/platform/plugins/shared/inference/server/chat_complete/anonymization/regex_worker_wrapper.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+const { workerData } = require('worker_threads');
+
+if (process.env.NODE_ENV !== 'production') {
+  /* eslint-disable @kbn/imports/no_boundary_crossing */
```

Comment:
ah this disables it for the whole file because it's a multiline comment. that's fine but can you move it to the top?